### PR TITLE
Add processor for new Koschei message topic

### PR DESCRIPTION
**Pull request by mizdebsk**
Koschei recently added a new message topic in https://github.com/msimacek/koschei/issues/17 -- `koschei.collection.state.change`

### CODE REVIEW

File: fedmsg_meta_fedora_infrastructure/koschei.py [hunk 1]
```diff
@@ -47,6 +47,20 @@ def subtitle(self, msg, **config):
             return info.format(name=content['name'],
                                collection=collection,
                                koji_instance=content['koji_instance'])
+        elif 'koschei.collection.state.change' in msg['topic']:
```
Comment (bowlofeggs) [inline fedmsg_meta_fedora_infrastructure/koschei.py]: I suggest adding comments here about what this code block is for.

File: fedmsg_meta_fedora_infrastructure/koschei.py [hunk 2]
```diff
@@ -63,6 +77,10 @@ def link(self, msg, **config):
             if 'collection' in msg['msg']:
                 url += '?collection=' + msg['msg']['collection']
             return url
+        elif 'koschei.collection.state.change' in msg['topic']:
```
Comment (bowlofeggs) [inline fedmsg_meta_fedora_infrastructure/koschei.py]: Comments would be good here too. When does this message happen?

Comment (mizdebsk) [inline fedmsg_meta_fedora_infrastructure/koschei.py]: I thought that the topic is self-descriptive and it matches generic pattern `app.something.state.change` - it is sent when state of collection in [Koschei](https://fedoraproject.org/wiki/Koschei) changes. But sure, I'll add comment describing format and purpose of the message.

**Comment by pypingou** [comment]
Do we want to add the two comments and merge ?

**Comment by pypingou** [comment]
@bowlofeggs I let you merge this one, it looks fine to me

**Comment by pypingou** [comment]
@mizdebsk could you check the tests?

**Comment by mizdebsk** [comment]
@pypingou Build failure on Travis CI does not seem to be related to this pull request. Note that the original build for commit 24edd56 succeeded.

**Comment by pypingou** [comment]
Do we know if there is a way to ask travis for a new build?

**Comment by mizdebsk** [comment]
I've rebased this PR, which should trigger new Travis build.

**Comment by pypingou** [comment]
Perfect, let's merge then ! :)

**Pull request merged**